skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
will do. @mortonjt , could you comment on if you made any change to the bxpython original _intersection.pyx?